src/util/clipboard.cpp

```diff
@@ -15,6 +31,11 @@ void Clipboard::add(const QUrl& url) {
 void Clipboard::finish() {
     QMimeData* data = new QMimeData;
     data->setUrls(instance()->m_urls);
+#ifdef __LINUX__
```

Comment:
The limitation on linux is a bit confusing here because it depends on the file manager not on the OS

Reply:
It would not break anything to make it available for all targets. But is there a real use case for it?

Reply:
For me it's not about the real use case, it's more about the difference between the OS targets caused by a file-manager difference that could in theory exist on all platforms